packages/playground/src/utils/validators.ts

```diff
@@ -777,3 +777,9 @@ export async function isAvailableName(grid: GridClient, name: string) {
     return { message: "Name is already taken." };
   }
 }
+
+export function validateSmtp(v: string) {
+  const emailValidation = isEmail("Please provide a valid email address.")(v);
+  const username = isAlpha("Name must start with an alphabetical character.")(v[0]);
+  return emailValidation && username;
```

Comment:
What if the user entered a username, not an email? This validation would expect the user to always to enter an email.

Reply:
No, it will expect both of them you can try it